spec/requests/children_api_spec.rb

```diff
@@ -44,6 +44,20 @@
           deleted_folder,
           other_folder
         ] }
+
```

Comment:
can this be turned into a shared_example?

Reply:
Yeah, I was waffling between shared_context and shared_examples for this bit. I landed on not refactoring due to decision paralysis. If this feature sticks around and is expanded to other endpoints, I'll definitely refactor it and put it with the other request shared examples.